sdcm/provision/aws/capacity_reservation.py

```diff
@@ -0,0 +1,241 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2024 ScyllaDB
+import logging
+import time
+from datetime import datetime, timedelta
+from typing import List, Dict, Tuple
+
+from botocore.exceptions import ClientError
+import boto3
+
+LOGGER = logging.getLogger(__name__)
+
+
+# Shlould runner be created in the same az always? - in that case CR needs to be created before sct-runner $0.01/GB data transfer price
+
+
+class CapacityReservationError(Exception):
```

Comment:
We have `sdcm/exceptions.py` module for such things.

Reply:
fixed